web/help/GuideUnit1.md

```diff
@@ -186,7 +187,7 @@ Try these examples to learn more:
     the computer will be confused and think you're defining a new
     variable.  This can cause a `Parse error` message.
 
-Once you've understood these examples, try your own combinations as well.
+Once you've understood these examples, try changing the numbers or making other changes.
```

Comment:
Hmm.  I'm usually not a fan of the "change some numbers" suggestion, because it's giving students an easy out to just do random stuff without any specific intention, and check off the box.  I very much want students to add their own shapes, and hopefully even imagine and try to reproduce some design.

This might be a two-step process, though.  I think it would be an improvement if you ask them to just try some modifications as you've done here, but then follow up with an explicit exercise to do something more deliberate.  The exercise should have three steps: (1) Reflect on what you know how to do (answer: draw text, and different sized circles and rectangles, but only at the center of the screen., (2) Draw a name tag on paper using only these kinds of shapes, (3) create a program that displays a picture of what you drew.  If I were teaching, I'd have students submit their paper drawings (or photos thereof, given COVID stuff) as well as their programs, to verify that they made a goal and decided how to accomplish the goal, instead of just doing things at random.